### PR TITLE
feat: MySQL Connector/Python example add some log to give some instructions

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the code of `mysql_connector_python_example.py` to keep it consistent with https://github.com/pingcap-inc/tidb-example-python/pull/1.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14522
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-sample-application-python-mysql-connector.md [hunk 1]
```diff
@@ -151,9 +151,12 @@ def simple_example() -> None:
 
             # create players with bulk inserts.
             # insert 1919 players totally, with 114 players per batch.
-            # each player has a random UUID
+            # all players have random uuid
+            print(f'start to insert one by one, it will take a long time')
             player_list = random_player(1919)
             for idx in range(0, len(player_list), 114):
+                if idx % 100 == 0:
+                    print(f'inserted {idx} players')
```
Comment (wd0517) [inline develop/dev-guide-sample-application-python-mysql-connector.md:159]: it not work

Comment (Icemap) [reply develop/dev-guide-sample-application-python-mysql-connector.md:159]: Got it.

**Review by wd0517 — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**